### DISCUSSION TITLE
Beginner tutorial and examples

### DISCUSSION THREAD

**Discussion by kevinqtogitty** [pmndrs/react-spring#2022] [Show and tell]
For someone who's relatively new to programming (~7 months) reading the react-spring docs went way over my head a few months back. So I spent his weekend learning each hook. I've made a repo with tutorials walking through every step on how to use each one with very basic examples. 

Love to hear some feedback, especially if something is explained incorrectly :)

Happy springing!

https://github.com/kevinqtogitty/Im-SPRUNG-react-spring-tutorial

**Comment by joshuaellis** [comment]
Sorry to hear you've struggled to work with our documentation, have you seen the `beta` docs? – https://beta.react-spring.dev/docs I believe them to be better written for starters, so it'd be great to get your feedback if you're still unsure about some parts.

**Reply by kevinqtogitty** [reply]
Yes! I used the beta docs to make this. They are much clearer especially for when you just want to learn how the hooks work on a basic level. I haven't gone through everything yet, but definitely plan to. Hope to make some fun stuff to submit as an example on the new docs!
